build/storage/tests/it/ptf_tests/README.md

```diff
@@ -0,0 +1,8 @@
+# Description
+PTF tests are integration tests based on PTF library. This library was chosen for the matter of unification of testing framework between development teams. 
```

Comment:
No need to put this kind of internal information here (i.e. unification of testing frameworks). Focus on PTF tests description and a link to get more information on that.

Reply:
done